src/views/OtherStoresInventoryModal.vue

```diff
@@ -58,9 +59,28 @@ export default defineComponent({
     IonToolbar 
   },
   props: ["otherStoresInventory"],
+  data() {
+    return{
+      queryString: "",
+      filteredInventory: [] as any
+    }
+  },
+  mounted() {
+    // Create a copy of otherStoresInventory on mount
+    this.filteredInventory = this.otherStoresInventory.slice();
+  },
   methods: {
     closeModal() {
       modalController.dismiss({ dismissed: true });
+    },
+    searchFacilities(){
+      if (this.queryString !== "") {
+        this.filteredInventory = this.otherStoresInventory.filter((facility: any) =>
+          facility.facilityName.toLowerCase().includes(this.queryString.toLowerCase()));
```

Comment:
This should be in single line as having this in multiple lines will creates confusion regarding the return value and thus reduces the code readability.